app/router.js

```diff
@@ -120,6 +120,9 @@ router.map(function() {
     this.route('users', function() {
       this.route('list', { path: '/:users_status' });
       this.route('user', { path: '/u/:user_id' });
```

Comment:
@ritikamotwani use this route. This already accepts a `user_id`. 

This is exactly what is needed. Change any other route as needed to make the structure as follows.

`/admin/users/:user_status` - List of users
`/admin/users/:user_id` - Viewing a specific user
`/admin/users/:user_id/sessions/:session_status` - Listing sessions that belong to a user. 

For the first two, checkout `events.view` and `events.list` for inspiration on how to go about it.